### PR TITLE
feat(ui): Edit Stage / Manifest Preview

### PR DESCRIPTION
**Pull request by rpelczar**
https://github.com/akuity/kargo/assets/7262558/84788edd-8a8c-41e5-9e87-5826e95123b3

### CODE REVIEW

File: ui/src/features/stage/stage-details.tsx [hunk 1]
```diff
@@ -92,7 +51,21 @@ export const StageDetails = () => {
...
+              defaultActiveKey='1'
+              items={[
+                {
+                  key: '1',
+                  label: 'Available Freight',
+                  children: <AvailableFreight stage={data.stage} onSuccess={refetch} />
+                },
+                {
+                  key: '2',
+                  label: 'Manifest Preview',
```
Comment (jessesuen) [inline ui/src/features/stage/stage-details.tsx:64]: No need for the word "preview". I suggest:

```suggestion
                  label: 'Manifest',
```

**Review by jessesuen — COMMENTED** [review]
@devholic do we know why apiVersion and kind are missing from the live object?

File: ui/src/features/stage/stage-details.tsx (hunk 1 above)
Comment (jessesuen) [reply ui/src/features/stage/stage-details.tsx:64]: Or Live Manifest (which is what we use in Argo CD)

**Comment by jessesuen** [comment]
@devholic this PR got closed because the other PR said "Resolves this one" and so I had to reopen it.

**Review by rbreeze — APPROVED** [review]
LGTM!

**Pull request merged**